cl/recap/tasks.py

```diff
@@ -1429,12 +1434,56 @@ def mark_fq_status(fq, msg, status):
     fq.save()
 
 
+@app.task(bind=True, max_retries=5, interval_start=5 * 60)
+def send_docket_to_webhook(d_pk: int, since: datetime, epq_pk: int) -> None:
+    d = Docket.objects.get(pk=d_pk)
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket=d
+    )
+    if docket_entries.count() == 0:
+        # No new docket entries.
+        return
+
+    epq = EmailProcessingQueue.objects.get(pk=epq_pk)
+    webhooks = Webhook.objects.filter(
+        event_type=WebhookEventType.RECAP_EMAIL,
+        user__profile__recap_email__in=epq.destination_emails,
+    )
+
+    serialized_docket_entries = []
+    for de in docket_entries:
+        serialized_docket_entries.append(DocketEntrySerializer(de).data)
+
+    for webhook in webhooks:
+        if webhook.enabled:
+            post_content = {
+                "webhook": {
+                    "event_type": webhook.event_type,
+                    "version": webhook.version,
+                    "date_created": webhook.date_created,
+                },
+                "results": serialized_docket_entries,
+            }
+            response = requests.post(
+                webhook.url, data=post_content, timeout=30
```

Comment:
```suggestion
                webhook.url, data=post_content, timeout=2
```

This needs to be faster. If their server goes down and we get a lot of content for it, we'll be hurting if all our threads are tied up for 30s each waiting for a response.